configuration/src/test/java/com/synopsys/integration/configuration/property/PropertyTestHelpUtil.java

```diff
@@ -84,6 +84,7 @@ private PropertyTestHelpUtil() {
         if (expectedExampleValues != null) {
             Assertions.assertNotNull(property.listExampleValues(), String.format("A %s property has a limited number of values that should be described.", property.describeType()));
             Assertions.assertTrue(CollectionUtils.containsAll(expectedExampleValues, property.listExampleValues()), String.format("The %s provided unexpected example values.", property.getClass().getSimpleName()));
+            Assertions.assertTrue(CollectionUtils.containsAll(property.listExampleValues(), expectedExampleValues), String.format("The %s provided unexpected example values.", property.getClass().getSimpleName()));
```

Comment:
It is different - I assumed this util was doing an equals check but had one slip through - I had copied another properties help assertion and it passed because given expected values of "ONE, TWO" and an actual of "TWO", CollectionUtils.containsAll(expected, actual) would return true. All of the actual are in the expected. This double invert assert is closer to an equals where each list must contain the other. Ideally I would want CollectionUtils.areEqual() but I did not see it so I opted for this as it's closer to an equals.

Reply:
I see, could you put a short comment there?